hw/ip/i2c/doc/_index.md

```diff
@@ -25,14 +25,15 @@ See that document for integration overview within the broader top level system.
 - Support for the following optional capabilities:
     - Clock stretching in the host mode
     - Automatic clock stretching in the target mode when TX FIFO is empty during a read<sup>2</sup>
+    - Clock stretching in the target mode after any complete transaction
```

Comment:
just FYI, i missed one of the latest comments in the doc, and there may need to be a distinction on this behavior whether it's read / write.
I'm trying to confirm, will update once we know more.